build.gradle

```diff
@@ -108,6 +108,10 @@ spotbugsMain {
 	}
 }
 
+spotbugs {
+	excludeFilter = file("${rootDir}/config/spotbugs/exclude.xml")
```

Comment:
As noted in #11, this can't be done because single quotations treats the `{rootDir}` variable as a string literal instead of a value that needs to be evaluated then converted into a string